.ci-mgmt.yaml

```
@@ -39,3 +39,4 @@ releaseVerification:
   python: examples/bucket-py
   dotnet: examples/serverless-cs
   go: examples/bucket-go
+AutoMergeProviderUpgrades: false
```

```suggestion
autoMergeProviderUpgrades: false
```

Is it case sensitive?

---

yeah, thanks for spotting!